src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const treeItems = document.querySelectorAll('.tree li');
+
+  treeItems.forEach((li) => {
+    const childUl = li.querySelector('ul');
+
+    if (childUl) {
+      const span = document.createElement('span');
+
+      span.textContent = li.firstChild.textContent.trim();
```

Comment:
Ensure that `li.firstChild` is a text node before accessing `textContent`. If the first child is not a text node, this could lead to errors or unexpected behavior. Consider checking `li.firstChild.nodeType === Node.TEXT_NODE` before accessing `textContent`.